README.rst

```diff
@@ -1,19 +1,111 @@
+####################
 edx-name-affirmation
+####################
+
+| |Build Status| |Coveralls|
+
+This library contains data model and business logic to store verified 
+personal information for learners of the edx-platform.
+This library is designed to be a pluggable library to the edx-platform.
```

Comment:
I received a question about how to disable the plugin. Can you either link to some plugin docs or explain how an installation would disable this plugin?

Reply:
That make sense. I can talk about it a bit.